Userland/Utilities/file.cpp

```diff
@@ -19,59 +20,59 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
-static Optional<DeprecatedString> description_only(DeprecatedString description, [[maybe_unused]] DeprecatedString const& path)
+static ErrorOr<Optional<String>> description_only(StringView const& description, [[maybe_unused]] StringView const& path)
```

Comment:
Never pass string views by const reference, always by value

Reply:
For reference: https://github.com/SerenityOS/serenity/commit/8b1108e4858f797c9216dc8ae4a3918ad50c73b4